src/checkstatus/SSMAutomationExecutionStatusWaiter.py

```diff
@@ -0,0 +1,31 @@
+import botocore
+import constants
+import waiter_acceptor_config
+
+def check_automation_execution_status(automation_execution_id, max_attempts, ssm):
```

Comment:
Can we add some logging to show the method's progress

Reply:
waiter execution does not return any values - other than when it completes so i dont think anything specific can be added.

Reply:
ok